openipam/api/serializers/network.py

```diff
@@ -155,6 +155,9 @@ def update(self, instance, validated_data):
         validated_data["changed_by"] = self.context["request"].user
         return super(NetworkToVlanSerializer, self).update(instance, validated_data)
 
+    def get_vlan(self, obj):
+        return obj.vlan_id
```

Comment:
should this be obj.pk or obj.id ?

Reply:
This reference is on the bridge table.  vlan_id allows us not to have to do nasty django joins.  It's the foreign key reference for id.  Otherwise it would have to be obj.vlan.pk or obj.vlan.id